tests/snap_test.go

```diff
@@ -190,17 +190,29 @@ func setupGPIO() error {
 }
 
 func TestBlinkOperation(t *testing.T) {
-	// test blink operation
+	/*
+		`test-blink` runs until it is stopped.
+		1. We use a context with a timeout to stop it after 5 seconds.
+		2. This does not work in a GitHub runner, so we also do a force kill after 10 seconds.
```

Comment:
Best not to include the code logic inside comments as it can easily go out of sync.
```suggestion
		1. We use a context with a timeout to stop it after a few seconds.
		2. This does not work in a GitHub runner, so have to kill it afterwards.
```
I don't think moving the comments to above helps. Why not explain 2 right where it is relevant? The reader may not go to the top of the function to read these comments.